lmcache/v1/config.py

```diff
@@ -113,11 +113,15 @@ def _parse_quoted_string(value: str) -> str:
     "nixl_role": "pd_role",
     "controller_url": "controller_pull_url",
     "lmcache_worker_port": "lmcache_worker_ports",
+    "plugin_locations": "runtime_plugin_locations",
```

Comment:
This is a good practice which should be added to the `Best practice of LMCache development` which proposed in https://github.com/LMCache/LMCache/issues/2060